watcher/dockerfiles/resource_list.json

```diff
@@ -1,10 +1,7 @@
 [
     {
+        "Group": "batch",
```

Comment:
You can watch whatever K8s resources you want. Originally I think @zhenghuiwang selected Pods and Services for no particular reason.  In this PR i changed it to be a K8s job because i want to show you could submit a batch job and then collect metadata for it via annotations.